lib-ee/emqx_license/include/emqx_license.hrl

```diff
@@ -0,0 +1,38 @@
+%%--------------------------------------------------------------------
+%% Copyright (c) 2021 EMQ Technologies Co., Ltd. All Rights Reserved.
+%%
+%% @doc EMQ X License Management CLI.
+%%--------------------------------------------------------------------
+
+-ifndef(_EMQX_LICENSE_).
+-define(_EMQX_LICENSE_, true).
+
+-define(EVALUATION_LOG,
+    "\n"
+    "===============================================================================\n"
+    "This is an evaluation license that is restricted to 10 concurrent connections.\n"
+    "If you already have a paid license, please apply it now.\n"
+    "Or you could visit https://www.emqx.io/licenses to get a trial license.\n"
+    "===============================================================================\n"
+    ).
+
+-define(EXPIRY_LOG,
+    "\n"
+    "======================================================\n"
+    "Your license has expired.\n"
+    "Please visit https://www.emqx.io/licenses or\n"
```

Comment:
```suggestion
    "Please visit https://www.emqx.com/licenses or\n"
```

Reply:
Seems that https://emqx.com/apply-licenses/emqx is the address for obtaining licenses.